perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/FacilitiesManagerMethod.java

```diff
@@ -94,6 +94,22 @@ public Facility call(ApiCaller ac, Deserializer parms) throws PerunException {
 	},
 
 	/*#
+	 * Searches for the Facilities by theirs attribute.
```

Comment:
Please state here, that large attributes are not supported, and that only def and opt attribute types are supported.